### DISCUSSION TITLE
Just a quick question.

### DISCUSSION THREAD

**Discussion by ARAHTHUL** [Abdelrhman-AK/WinPaletter#8] [Q&A] [answered]
Hello! I just discovered your app and this looks promising, no joke!

Some time ago, I asked a question on Reddit, but no one answered, and I'll try again here; from the screenshots I sort of get an answer but I think you know better if I'm able to do it or not.

So, the question: is there a way to maintain the color I choose as accent throughout the system, but the title bars to not be changed? I want windows accent to be blue but title bars to be black.

Thank you in advance and have a good one!

**Comment by Abdelrhman-AK** [comment, marked as answer]
**You're welcome!**
When you change color from Windows Settings, it will overwrite the previous applied palette. So everytime you change color from Windows Settings you should open the app and change titlebar's color manually.

**Reply by ARAHTHUL** [reply]
I actually managed to do it with the app!
Thank you very much again! Have a good one kind sir/ma'am!
